### PR TITLE
Remove dependency on collective.oembed

### LINKED ISSUE #13: Remove dependency on collective.oembed
To avoid problems like the one reported on #12.

We need to find out a well maintained Python library for this as [python-oembed](https://pypi.python.org/pypi/python-oembed) seems to be abandoned.

### PR DESCRIPTION
**Pull request by hvelarde**
closes #13

### CODE REVIEW

File: src/sc/embedder/content/embedder.py [hunk 1]
```diff
@@ -18,9 +17,10 @@
...
 from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
 from sc.embedder import MessageFactory as _
+from sc.embedder.interfaces import IConsumer
 from z3c.form import button
-from z3c.form.interfaces import IFormLayer
 from z3c.form.interfaces import IFieldWidget
+from z3c.form.interfaces import IFormLayer
 from z3c.form.widget import FieldWidget
 from zope import schema, component
 from zope.component import adapter
```
Comment (rodfersou) [inline src/sc/embedder/content/embedder.py]: How about to follow plone.api conventions here http://docs.plone.org/external/plone.api/docs/contribute/conventions.html#grouping-and-sorting ?

Comment (hvelarde) [inline src/sc/embedder/content/embedder.py]: I'm following the conventions: https://github.com/plone/plone.api/issues/187

and remember: never ever mix the implementation of a feature with code analysis stuff for the sake of the code reviewers mind.

File: src/sc/embedder/utility.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf-8 -*-
+"""This code was taken from collective.oembed and is included here just
+to remove the dependency on that package.
+
+See: https://github.com/simplesconsultoria/sc.embedder/issues/13
+
+TODO: Refactor it to get the list of providers on an alternative way.
+      The Iframely projet maintains a list of oEmbed providers updated.
```
Comment (rodfersou) [inline src/sc/embedder/utility.py]: typo: project ?

File: src/sc/embedder/content/embedder.py (hunk 1 above)
Comment (rodfersou) [inline src/sc/embedder/content/embedder.py]: sorry, I didn't see all the source.. just the github view

**Pull request merged**